### ISSUE TITLE
Wechaty introduction video discussion (2024)

### ISSUE THREAD

**Issue by aherman3** [wechaty/docusaurus#1536]
Hello everyone!

This issue has been opened for community discussion on Wechaty's short introduction video that will be featured on the site. 

This video aims to be a less than 1 minute introduction to what Wechaty is, its services, and unique strengths. The project is currently under development and open for any comments to help improve its quality. 

- ~Here is the first draft video [storyboard](https://drive.google.com/file/d/1j4A2-NPHKBY5TjA_49o17HYJGnY1JNp2/view?usp=sharing)~
- ~2nd draft [video storyboard](https://drive.google.com/file/d/1DxK4Yd9CO3olWjq2aRadOD7IMrUKJXVB/view?usp=sharing)~
- **The most recent [video storyboard](https://drive.google.com/file/d/1Z19wY8-PnnMnQ4HFgaEkTFgQqDpGiVgh/view)**

Everyone is welcome to take a look and leave helpful suggestions.

**Comment by aherman3** [comment]
I have already suggested the following changes directly to the production team:

- Slides 3, 4: Please use the green bot from the Wechaty logo instead. 
- Slides 5, 6: Add graphics displaying functionings being pointed to outside the phone. For an idea of what graphics I mean, refer to 00:28 of the [previous draft video](https://vimeo.com/user25098419/review/578811418/4b79b846fb) 
- Slide 8: With the added graphics on 5 and 6, I don’t think this slide will be needed, and its voiceover text can be combined with slide 7.
- Slide 11: Add a graphic of a globe with connected peoples 
- Slide 12: Behind the graphic, add bubbles with the 6 lines of code, (refer to [this image](https://drive.google.com/file/d/1Y-jC2f_k_mrgRjgTg98h25TlIuL6GX0b/view)). Also, remove the robot graphic. 
- Slide 13: This video will be featured on our website, so displaying the website page is a bit strange. Perhaps replace this with the Wechaty logo on the computer screen (like slide 9) 
- Slide 14: Beneath the current text, add Wechaty’s contact information

**Comment by huan** [comment]
@aherman3 Thanks for your contribution to our landing video!

I love your first change request: we plan to remove this ugly avatar from our Wechaty community in the near future:

<img width="162" alt="image" src="https://github.com/wechaty/docusaurus/assets/1361891/d2673bd6-9148-4a02-9613-dda58ffd67b1">

**Comment by aherman3** [comment]
The video storyboard has been updated according to the above comments. Take a look [here](https://drive.google.com/file/d/1DxK4Yd9CO3olWjq2aRadOD7IMrUKJXVB/view?usp=sharing). 

@huan the avatar on slide 10 hasn't been removed yet. I'll pass on your request

**Comment by aherman3** [comment]
> The video storyboard has been updated according to the above comments. Take a look [here](https://drive.google.com/file/d/1DxK4Yd9CO3olWjq2aRadOD7IMrUKJXVB/view?usp=sharing).
> 

In response to this updated storyboard, I passed on the following suggestions
- Slide 4: Please space the logos to make a more complete circle.
- Slide 9: Put the text to the side of the globe graphic, as in 00:35 of the [previous video](https://vimeo.com/user25098419/review/578811418/4b79b846fb#)
- Slide 10: We plan to phase out this bot avatar so remove it completely.
- Let’s replace the laptop in Slide 7 with the smartphone graphic used in the first storyboard.

This list is not absolute and the storyboard is still open to more input!

**Comment by aherman3** [comment]
The [storyboard](https://drive.google.com/file/d/1Z19wY8-PnnMnQ4HFgaEkTFgQqDpGiVgh/view?usp=sharing) has been updated again!

**Comment by huan** [comment]
Can we try to update the script by aligning the following Wechaty elevator pitch?

> Wechaty is a conversational RPA SDK that simplifies the process of building chatbots. It's like a Swiss Army knife for chatbot development, providing a universal interface to various messaging platforms such as WhatsApp, WeChat, and more. With Wechaty, you write your bot code once, and it runs on any of the supported platforms. This means you can focus on creating engaging conversational experiences without worrying about the underlying complexities of each platform's API. It's open-source, easy to use, and backed by a vibrant community that's ready to help you bring your chatbot ideas to life.

**Comment by aherman3** [comment]
sure @huan ! I made a table with updated lines compared to the original. I aimed to align the language with the elevator pitch, but keep the meaning the same so the current graphics aren't greatly affected. Changed lines are *italicized*

| Slide # | Original | Updated |
| ------- | ------- | ------- |
| 1 | Feeling overwhelmed by the endless stream of messages? | Feeling overwhelmed by the endless stream of messages? |
| 2 | Say hello to Wechaty – the ultimate solution for chatbot developers! | Say hello to Wechaty – the ultimate solution for chatbot developers! |
| 3 | Harnessing the power of Robotic Process Automation (RPA),  | *Wechaty is a conversational RPA SDK that simplifies the process of building chatbots,* |
| 4 | Wechaty builds an API layer atop your favorite messaging platforms, so you can build a personalized chatbot with ease. | *providing a universal interface to various messaging platforms including WhatsApp, WeChat, and more.* |
| 5 | Imagine effortlessly managing contacts, automating messages,  | *Write your bot code once to effortlessly manage contacts, automate messages* |
| 6 | and customizing interactions to suit your needs – all with Wechaty's intuitive solutions. | *and customize interactions — without worrying about complex API systems.* |
| 7 | Save time, enhance productivity, and unlock endless possibilities with Wechaty. | **Delete** |
| 8 | Ready to embark on your journey with Wechaty?  | Ready to embark on your journey with Wechaty?  |
| 9 | Join our international community of developers and enthusiasts today. | Join our international community of developers and enthusiasts today.  |
| 10 | Getting started only takes 6 lines of code, in whichever programming language you prefer. | Getting started only takes 6 lines of code, in whichever programming language you prefer. |
| 11 | Wechaty. Where innovation meets simplicity, revolutionizing messaging one line of code at a time. | *Code less, and work more, with Wechaty.* |
| 12 | Get in touch with us today to learn more. | Get in touch with us today to learn more. |

**Comment by huan** [comment]
It looks perfect, Annie!

I want to confirm that there will be a URL call for action will be the ending screen of the video, correct? How long will it stay?

And which URL do you think is better: https://github.com/wechaty/wechaty or https://wechaty.js.org ? And why?

Thank you!

**Comment by aherman3** [comment]
> It looks perfect, Annie!
> 
> 
> 
> I want to confirm that there will be a URL call for action will be the ending screen of the video, correct? How long will it stay?
> 
> 
> 
> And which URL do you think is better: https://github.com/wechaty/wechaty or https://wechaty.js.org ? And why?
> 
> 
> 
> Thank you!

Yes! The last slide lists some Wechaty contact info: the github link, google groups, and discord server QR code. Since this video is planned to be posted on the wechaty.js.org homepage, I don't think it's necessary to provide this link.

**Comment by huan** [comment]
I think the GitHub link would be enough because the call to action should be concise and not require any choice.

What do you think? Should we keep only one action to the watchers?

**Comment by aherman3** [comment]
> I think the GitHub link would be enough because the call to action should be concise and not require any choice.
> 
> 
> 
> What do you think? Should we keep only one action to the watchers?

I agree just the GitHub link is best. The discord QR is already listed lower on the homepage. I'll forward that change + the new script

**Comment by huan** [comment]
Sounds perfect. Thank you very much, Annie!
